docs/api/core/commands/world-commands/entity-query.md

```diff
@@ -25,23 +25,11 @@ code {
 
 <table>
 <tr>
-<td style="padding: 14px; border: none; width: 23ch"><a href="{{urlRoot}}/api/core/commands/world-commands/entity-query/command-component-manager">CommandComponentManager</a></td>
-<td style="padding: 14px; border: none;"></td>
-</tr>
-<tr>
-<td style="padding: 14px; border: none; width: 23ch"><a href="{{urlRoot}}/api/core/commands/world-commands/entity-query/command-responses">CommandResponses</a></td>
-<td style="padding: 14px; border: none;">ECS component contains a list of CreateEntity command responses received this frame. </td>
-</tr>
-<tr>
-<td style="padding: 14px; border: none; width: 23ch"><a href="{{urlRoot}}/api/core/commands/world-commands/entity-query/command-sender">CommandSender</a></td>
-<td style="padding: 14px; border: none;">ECS component is for sending EntityQuery command requests to the SpatialOS runtime. </td>
-</tr>
-<tr>
-<td style="padding: 14px; border: none; width: 23ch"><a href="{{urlRoot}}/api/core/commands/world-commands/entity-query/received-response">ReceivedResponse</a></td>
+<td style="padding: 14px; border: none; width: 16ch"><a href="{{urlRoot}}/api/core/commands/world-commands/entity-query/received-response">ReceivedResponse</a></td>
 <td style="padding: 14px; border: none;">An object that is the response of an EntityQuery command from the SpatialOS runtime. </td>
 </tr>
 <tr>
-<td style="padding: 14px; border: none; width: 23ch"><a href="{{urlRoot}}/api/core/commands/world-commands/entity-query/request">Request</a></td>
+<td style="padding: 14px; border: none; width: 16ch"><a href="{{urlRoot}}/api/core/commands/world-commands/entity-query/request">Request</a></td>
```

Comment:
hm

Reply:
Wat

Reply:
Ah wait I know what this is. We calculate the width of these elements based on the contents. Some stuff was removed from the table so it altered the width.